diyspk/diskutils/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = diskutils
-SPK_VERS = 1.42.6
-SPK_REV = 1
+SPK_VERS = 1.45.4
+SPK_REV = 2
```

Comment:
Please replace installer.sh and dsm-control.sh with a single service-setup.sh
https://github.com/SynoCommunity/spksrc/pull/3795/files#diff-dcc938b42979475c6f97ea210865109eR19-R22